pkg/ebpf/tracee.go

```diff
@@ -1626,6 +1630,65 @@ func (t *Tracee) triggerSeqOpsIntegrityCheckCall(
 	return nil
 }
 
+// triggerMemDump is used by a Uprobe to trigger an eBPF program
+// that prints the seq ops pointers
+func (t *Tracee) triggerMemDump() error {
+	_, ok := t.events[events.PrintMemDump]
+	if !ok {
+		return nil
+	}
+
+	printMemDumpFilters := t.config.Filter.ArgFilter.GetEventFilters(events.PrintMemDump)
+	if printMemDumpFilters == nil {
+		return fmt.Errorf("no address or symbols were provided to print_mem_dump event. " +
```

Comment:
@rafaeldtinoco looks like this event will also be configured through filters.
In the context of this PR we need to document it, and exclude it from the filtering at the derivation stage later.

Reply:
I also think this is more support for separating event configuration and filtering

Reply:
Okay, so, for this PR, it should be rebased to include the same skip as commit 12460c13 AND I do agree with Nadav, we should create an interface for providing config options to signatures (instead of using the filters themselves).

Reply:
I suggest we open an issue on this and create this interface in a new PR.
For now @AsafEitani you should use the skip mechanism that @rafaeldtinoco did for previous similar events.

Reply:
Done